k9mail/src/main/java/com/fsck/k9/activity/setup/AccountSettings.java

```diff
@@ -739,11 +710,21 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
                 }
             });
 
-            mCryptoSupportSignOnly.setChecked(mAccount.getCryptoSupportSignOnly());
+            cryptoMenu.setOnPreferenceClickListener(null);
         } else {
-            final Preference mCryptoMenu = findPreference(PREFERENCE_CRYPTO);
-            mCryptoMenu.setEnabled(false);
-            mCryptoMenu.setSummary(R.string.account_settings_no_openpgp_provider_installed);
+            cryptoMenu.setSummary(R.string.account_settings_no_openpgp_provider_configured);
```

Comment:
See this explicitly binds 'crypto' to meaning OpenPGP. It's not very forward compatible.